typing/ikinds/ikinds.mli

```diff
@@ -47,16 +47,18 @@ val sub_or_error :
   ('l2 * Allowance.allowed) Types.jkind ->
   (unit, Jkind.Violation.t) result
 
+type lookup_result =
```

Comment:
(nit) I suggest _sweeksify_-ing this type. If that word is unfamiliar to you (I suspect it is), we can chat offline about it.